setup.py

```diff
@@ -45,7 +45,7 @@
     package_dir={"": "src"},
     py_modules=[splitext(basename(path))[0] for path in glob("src/*.py")],
     zip_safe=False,
-    install_requires=["typer", "mdformat-gfm"],
+    install_requires=["typer", "mdformat-gfm", "pydocstyle @ git+https://github.com/robocorp/pydocstyle"],
```

Comment:
If we go on this route, then please ensure we end up with our GitHub hosted **lazydocs** in the _devutils/poetry.lock_ file after `inv install`-ing it to see what it brings.

Then, any other package (library) pinning our local **devutils**, should bring in the env both **lazydocs** & **pydocstyle**, both as top-level development dependencies. (aka you have them in each env during `inv install/devinstall` but no trace of these in the published package as a library in PyPI)

----

I haven't tested it, but this should be reflected on your side before merging.